bigtable/cmd/cbt/cbt.go

```diff
@@ -420,8 +420,17 @@ var commands = []struct {
 			"  batch-size=<500>                      The max number of rows per batch write request\n" +
 			"  workers=<1>                           The number of worker threads\n\n" +
 			"  Import data from a csv file into an existing cbt table that has the required column families.\n" +
-			"  See <example.csv.github.com/cbt-import-sample.csv> for a sample .csv file and formatting.\n" +
-			"  If no column family row is present, use the column-family flag to specify an existing family.\n\n" +
+			"  The first row should have a blank cell (this column is for row keys), then hold the column names.\n"
```

Comment:
Put the column family example first because it uses fewer parameters.
 
`The CSV file should contain two rows of headers: the first for column families and the second for column qualifiers. The first column will be used for the rowkey, so it should be left blank. Each column families only needs to be included once. The remaining rows of the CSV should contain a rowkey and its cells.
`
The CSV file uses one row of headers specifying the column qualifiers and a blank for the rowkey. You can add an additional row of headers for the column families and then remove the column-family argument from the import command.